css-positioning/absolute.css

```diff
@@ -0,0 +1,51 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 15px;
+  right: 20px;
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+
+  /* your code here */
+  position: absolute;
+  bottom: -10px;
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 55%));
+
+  /* your code here */
+  position: absolute;
+
+  /* top:10; */
+  height: 300px;
+  width: 300px;
+  top: 10px;
+  right: 10px;
+  bottom: 10px;
+  left: 10px;
+}
```

Comment:
You don't need to use both height / width & offset properties. You can use one or the other. So in this instance, you can either keep your height & width & remove the offset properties, or you can remove the height & width

Reply:
![16578463543](https://user-images.githubusercontent.com/122063836/215189962-0be90e65-802b-42fe-be99-0815064ee067.PNG)